src/components/Timeline/TimelineBrowser.js

```diff
@@ -213,12 +214,57 @@ LegendAndBound.propTypes = {
   maxCount2SD: number.isRequired,
 };
 
+const RecordAndIntervalCount = ({
+  availableWidth,
+  recordCount,
+  intervalCount,
+  intervalsWithRecordsCount,
+}) => {
+  const recordCountLabel = `${recordCount} total records`;
+  const intervalCountLabel = intervalCount ? `${intervalCount} intervals, ${intervalsWithRecordsCount} have records` : '';
```

Comment:
TODO: ^this should use I18n.  

...manually parsing the count would be egregious.